src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Component.razor

```diff
@@ -105,6 +105,55 @@
     <BitButton Target="_self" Href="https://github.com/bitfoundation/bitframework" IsEnabled="false">
         <span>Bit Foundation From Span</span>
     </BitButton>
+
+    <hr />
+
+    <div>
+        <h3>Toggle Button</h3>
+        <h4>Checked Button</h4>
+        <BitToggleButton @bind-IsChecked="TogleButtonChecked"
+                         Label="@(TogleButtonChecked?"Button checked":"Button unchecked")"
+                         IconName=@(TogleButtonChecked?"Emoji2" :"EmojiNeutral")></BitToggleButton>
+
+        <h4>Toggle Button with Onclick event </h4>
+        <BitToggleButton @bind-IsChecked="TogleButtonChecked"
+                         Label="@(TogleButtonChecked?"Button checked":"Button unchecked")"
+                         IconName=@(TogleButtonChecked?"Emoji2" :"EmojiNeutral")
+                         OnClick="ToggleButtonClick"></BitToggleButton>
+        <span>Check status is: @OnToggleButtonChecked</span>
```

Comment:
this sample is not appropriate, please consider changing it into a more clear and robust sample, for example, use OnChagne event instead